pkg/providers/vsphere/vmprovider_vm.go

```diff
@@ -1633,8 +1746,6 @@ func (vs *vSphereVMProvider) vmCreateDoPlacement(
 		}
 	}
 
-	pkgcnd.MarkTrue(vmCtx.VM, vmopv1.VirtualMachineConditionPlacementReady)
```

Comment:
Why is this removed again? Was it relocated?

Reply:
Yep, it's called in the defer statement in `vmCreateDoPlacement()` to avoid duplicate code for group placement.